### PR TITLE
Update the hyperlink address of High-Availability Clusters

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref #27094

### CODE REVIEW

File: content/zh/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ the same machine, and do not run user containers on this machine. See
  -->
 控制平面组件可以在集群中的任何节点上运行。
 然而，为了简单起见，设置脚本通常会在同一个计算机上启动所有控制平面组件，并且不会在此计算机上运行用户容器。
-请参阅[构建高可用性集群](/zh/docs/setup/production-environment/tools/kubeadm/high-availability/)
+请参阅[构建高可用性集群](/docs/setup/production-environment/tools/kubeadm/high-availability/)
```
Comment (jihoon-seo) [inline content/zh/docs/concepts/overview/components.md:65]:
```suggestion
请参阅[构建高可用性集群](/zh/docs/setup/production-environment/tools/kubeadm/high-availability/)
```

File: content/ja/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Kubernetesをデプロイすると、クラスターが展開されます。
 コントロールプレーンコンポーネントは、クラスターに関する全体的な決定(スケジューリングなど)を行います。また、クラスターイベントの検出および応答を行います(たとえば、deploymentの`replicas`フィールドが満たされていない場合に、新しい {{< glossary_tooltip text="Pod" term_id="pod">}} を起動する等)。
 
 コントロールプレーンコンポーネントはクラスター内のどのマシンでも実行できますが、シンプルにするため、セットアップスクリプトは通常、すべてのコントロールプレーンコンポーネントを同じマシンで起動し、そのマシンではユーザーコンテナを実行しません。
-マルチマスター VMセットアップの例については、[高可用性クラスターの構築](/docs/admin/high-availability/) を参照してください。
+マルチマスター VMセットアップの例については、[高可用性クラスターの構築](/docs/setup/production-environment/tools/kubeadm/high-availability/) を参照してください。
```
Comment (jihoon-seo) [inline content/ja/docs/concepts/overview/components.md:31]:
```suggestion
マルチマスター VMセットアップの例については、[高可用性クラスターの構築](/ja/docs/setup/production-environment/tools/kubeadm/high-availability/) を参照してください。
```

File: content/fr/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Ils détectent et répondent aux événements du cluster (par exemple, démarrer
 Les composants Master peuvent être exécutés sur n'importe quelle machine du cluster. Toutefois,
 par soucis de simplicité, les scripts de mise en route démarrent typiquement tous les composants master sur la
 même machine et n'exécutent pas de conteneurs utilisateur sur cette machine.
-Voir [Construire des Clusters en Haute Disponibilité](/docs/admin/high-availability/) pour une configuration d'exemple en multi-master-VM.
+Voir [Construire des Clusters en Haute Disponibilité](/docs/setup/production-environment/tools/kubeadm/high-availability/) pour une configuration d'exemple en multi-master-VM.
```
Comment (jihoon-seo) [inline content/fr/docs/concepts/overview/components.md:25]:
```suggestion
Voir [Construire des Clusters en Haute Disponibilité](/fr/docs/setup/production-environment/tools/kubeadm/high-availability/) pour une configuration d'exemple en multi-master-VM.
```

File: content/id/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ Komponen master dapat dijalankan di mesin manapun yang ada di klaster. Meski beg
 untuk memudahkan proses yang ada, <i>script</i> inisiasi awal yang dijalankan
 biasanya memulai komponen master pada mesin yang sama, serta tidak menjalankan
 kontainer bagi pengguna di mesin ini. Contoh konfigurasi <i>multi-master VM</i>
-dapat dilihat di modul [Membangun Klaster HA] (/docs/admin/high-availability/).
+dapat dilihat di modul [Membangun Klaster HA] (/docs/setup/production-environment/tools/kubeadm/high-availability/).
```
Comment (jihoon-seo) [inline content/id/docs/concepts/overview/components.md:33]:
```suggestion
dapat dilihat di modul [Membangun Klaster HA] (/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/id/docs/reference/glossary/kube-apiserver.md [hunk 5]
```diff
@@ -15,4 +15,4 @@ Komponen _control plane_ yang mengekspos API Kubernetes. Merupakan _front-end_ d
 
 <!--more-->
 
-Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/docs/admin/high-availability/).
+Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/docs/setup/production-environment/tools/kubeadm/high-availability/).
```
Comment (jihoon-seo) [inline content/id/docs/reference/glossary/kube-apiserver.md:18]:
```suggestion
Komponen ini didesain agar dapat diskalakan secara horizontal. Lihat [Membangun Klaster HA](/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/id/docs/setup/best-practices/multiple-zones.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ dukungan penuh federasi).
 satu Node master secara bawaan (_default_). Karena Service memerlukan 
 ketersediaan (_availability_) yang tinggi dan dapat mentolerir akan hilangnya sebuah zona, maka _control plane_
 diletakkan pada setiap zona. Pengguna yang menginginkan _control plane_ yang memiliki ketersediaan
-tinggi harus mengikuti instruksi [ketersediaan tinggi](/docs/admin/high-availability).
+tinggi harus mengikuti instruksi [ketersediaan tinggi](/docs/setup/production-environment/tools/kubeadm/high-availability/).
```
Comment (jihoon-seo) [inline content/id/docs/setup/best-practices/multiple-zones.md:77]:
```suggestion
tinggi harus mengikuti instruksi [ketersediaan tinggi](/id/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

File: content/fr/docs/reference/glossary/kube-apiserver.md [hunk 7]
```diff
@@ -15,5 +15,5 @@ tags:
 
 <!--more-->
 
-Il est conçu pour une mise à l'échelle horizontale, ce qui veut dire qu'il met à l'échelle en déployant des instances supplémentaires. Voir [Construire des Clusters en Haute Disponibilité](/docs/admin/high-availability/).
+Il est conçu pour une mise à l'échelle horizontale, ce qui veut dire qu'il met à l'échelle en déployant des instances supplémentaires. Voir [Construire des Clusters en Haute Disponibilité](/docs/setup/production-environment/tools/kubeadm/high-availability/).
```
Comment (jihoon-seo) [inline content/fr/docs/reference/glossary/kube-apiserver.md:18]:
```suggestion
Il est conçu pour une mise à l'échelle horizontale, ce qui veut dire qu'il met à l'échelle en déployant des instances supplémentaires. Voir [Construire des Clusters en Haute Disponibilité](/fr/docs/setup/production-environment/tools/kubeadm/high-availability/).
```

**Review by jihoon-seo — CHANGES_REQUESTED** [review]
Thank for your contribution.
I suggested some changes, since there are some translated versions of the target page. :)

**Comment by jihoon-seo** [comment]
- This PR resolves #27094

**Review by sftim — COMMENTED** [review]
@vaibhav2107 we try to limit fixes to a single language per PR - see https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr

This change would be difficult to approve as it requires co-ordination with more than 10 localization teams. Could you update just the English version for now?

Once the English change merges, we'd be very happy to review follow-up PRs to update the relevant localized pages.

**Comment by sftim** [comment]
This might be OK to merge but getting all the approvals will be tricky!

**Comment by seokho-son** [comment]
As a Korean contents owner, LGTM :) 
But I would like to postpone giving approval to this PR until @vaibhav2107  's feedback on the comment from @sftim .

**Comment by tengqm** [comment]
There is no change to zh localization although `language/zh` is added.
Good luck with this PR spanning so many teams.

**Comment by vaibhav2107** [comment]
> @vaibhav2107 we try to limit fixes to a single language per PR - see https://kubernetes.io/docs/contribute/new-content/overview/#languages-per-pr
> 
> This change would be difficult to approve as it requires co-ordination with more than 10 localization teams. Could you update just the English version for now?
> 
> Once the English change merges, we'd be very happy to review follow-up PRs to update the relevant localized pages.

Thank You @sftim for informing about this. Now on, I'll be careful not to fix more than one language in a PR.

**Comment by atoato88** [comment]
@vaibhav2107 
As @sftim says, could you update this PR to include changes for English contents only?
I think target files of this PR are following:
- content/en/docs/concepts/overview/components.md 
- content/en/docs/tasks/debug-application-cluster/debug-cluster.md

That's say remove other files (which include contents for other language localization) from this PR and those files will be updated in other PRs.

And also rebase is needed for current PR codeset.

**Comment by jihoon-seo** [comment]
The link in the English version was updated in the PR #26725.
This PR seems good to close.

**Comment by atoato88** [comment]
@vaibhav2107 
This PR has `content/en/docs/tasks/debug-application-cluster/debug-cluster.md` also.
So it's good to update PR with including `content/en/docs/tasks/debug-application-cluster/debug-cluster.md` only, and update files on multi language next on other PR.
WDYT?

**Comment by atoato88** [comment]
I think update flow is following:
- At first, this PR needs rebase with latest `kubernetes:master`.
- Next, update PR by excluding files except for `en` contents.
- Third, exclude `content/en/docs/concepts/overview/components.md` because #26725 fixed this file already.

**Comment by atoato88** [comment]
Thank you for updating. :smile: 
PR includes [`content/en/docs/concepts/overview/components.md` yet](https://github.com/kubernetes/website/pull/27139/files#diff-5aa7dbbe8bb73f90af59eb44d3118360ebe22fede8a50d99fedb3d9c197ef0f2R36-R41), so you need to exclude this file from PR.

**Comment by atoato88** [comment]
And I remove tags which doesn't related to current codeset.

/remove-language fr 
/remove-language id 
/remove-language it 
/remove-language ja 
/remove-language ko 
/remove-language pl 
/remove-language pt
/remove-language ru
/remove-language zh
/remove-language de
/remove-language es

**Pull request merged**